### PR TITLE
Document: podman was requested by local dev environment.

### PR DESCRIPTION
**Pull request by gyliu513**
We need to mention podman was requested by local development.

Fixed https://github.com/openshift/installer/issues/2508

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ For contributors who want to work up pull requests, the workflow is roughly:
 4. Push your changes to a topic branch in your fork of the repository.
 5. Make sure the tests pass, and add any new tests as appropriate.
 6. We run a number of linters and tests on each pull request.
-    You may wish to run these locally before submitting your pull request:
+    You may wish to run these locally before submitting your pull request (Make sure you have [podman](https://podman.io/getting-started/installation) installed):
```
Comment (abhinavdahiya) [inline CONTRIBUTING.md]: nit: update is the podman reference like https://github.com/openshift/installer/pull/2511/files#diff-6a3371457528722a734f3c51d9238c13R50

Comment (gyliu513) [reply CONTRIBUTING.md]: @abhinavdahiya not clear about your comments, the link you provided is the diff if my PR, and the link that I provided is for `podman` and it include many different arch support. Can you please show more detail for this?

Comment (abhinavdahiya) [reply CONTRIBUTING.md]: github doesn't automatically expand the view to jump to the linked line ;( .. if you expand lower on the diff view it should highlight the line.

the document uses `[test][link-anchor]` style links where the `link-anchor` is defined at the bottom of the document.

example: https://github.com/openshift/installer/blame/master/CONTRIBUTING.md#L50 and https://github.com/openshift/installer/blame/master/CONTRIBUTING.md#L100

Comment (gyliu513) [reply CONTRIBUTING.md]: Ah, I see, thanks @abhinavdahiya

**Pull request merged**